README.md

```diff
@@ -10,6 +10,15 @@
 [![Awesome Humane Tech](https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/humane-tech-badge.svg?sanitize=true)](https://github.com/humanetech-community/awesome-humane-tech)
 </div>
 
+<p align="center">
+  <span>English</span> |
+  <a href="https://github.com/LemmyNet/lemmy/tree/main/README.es.md">Español</a> |
+  <a href="https://github.com/alvarotrigo/fullPage.js/tree/master/lang/french#fullpagejs">Français</a> |
+  <a href="https://github.com/alvarotrigo/fullPage.js/tree/master/lang/russian#fullpagejs">Pусский</a> |
+  <a href="https://github.com/alvarotrigo/fullPage.js/tree/master/lang/chinese#fullpagejs">中文</a> |
+  <a href="https://github.com/alvarotrigo/fullPage.js/tree/master/lang/korean#fullpagejs">한국어</a>
+</p>
```

Comment:
Remove the ones that we dont support please. And I think we should put everything except English in a `readme` folder, otherwise the top level folder will have too many files. Finally, please use a relative link to the file, not via Github URL (eg `<a href="readmes/README.es.md">Español</a>`).